.github/workflows/helm-chart-release.yml

```diff
@@ -68,8 +68,12 @@ jobs:
         run: |
           make generate-images-file
 
+      - name: Generate policies files
+        run: |
+          make generate-policies-file
+
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.4.1
+        uses: jvanz/chart-releaser-action@main
```

Comment:
This should be change when this [fix](https://github.com/helm/chart-releaser/pull/235) is released.

Reply:
thanks for doing the fork!